projects/plugins/jetpack/extensions/blocks/sharing-button/sharing-button.php

```diff
@@ -66,37 +66,33 @@ function render_block( $attr, $content, $block ) {
 		return $content;
 	}
 
-	$service         = new $services[ $service_name ]( $service_name, array() );
-	$link_props      = $service->get_link(
+	$service      = new $services[ $service_name ]( $service_name, array() );
+	$link_props   = $service->get_link(
 		$post_id,
 		'share=' . esc_attr( $service_name ) . '&nb=1',
 		esc_attr( $data_shared )
 	);
-	$link_url        = $link_props['url'];
-	$link_classes    = sprintf(
+	$link_url     = $link_props['url'];
+	$link_classes = sprintf(
 		'jetpack-sharing-button__button style-%1$s share-%2$s',
 		$style_type,
 		$service_name
 	);
-	$link_aria_label = sprintf(
-		/* translators: %s refers to a string representation of sharing service, e.g. Facebook  */
-		esc_html__( 'Share on %s', 'jetpack' ),
+
+	$accessible_name = sprintf(
+		/* translators: %s refers to a string representation of sharing service, e.g. Facebook */
+		esc_html__( 'Click to share on %s', 'jetpack' ),
 		esc_html( $title )
 	);
 
-	$link_props = $service->get_link(
-		$post_id,
-		'share=' . esc_attr( $service_name ) . '&nb=1',
-		false,
-		esc_attr( $data_shared )
-	);
+	$accessible_name .= ' ' . __( '(Opens in new window)', 'jetpack' );
```

Comment:
I think the space should be part of the translatable string, so the thing can work on RTL languages.

Reply:
Good point, I see we did that in the Sharing_Source class for the $accessible_name as well. Added in 626e9fea9839b6c6617f80fec2be9c297eae5850